package/src/bin/omicron-package.rs

```diff
@@ -395,7 +395,8 @@ fn uninstall_all_packages(config: &Config) {
 
     // Once all packages have been removed, also remove any locally-stored
     // configuration.
-    std::fs::remove_dir_all(omicron_common::OMICRON_CONFIG_PATH).unwrap();
+    remove_all_unless_already_removed(omicron_common::OMICRON_CONFIG_PATH)
+        .unwrap();
 }
 
 fn remove_all_unless_already_removed<P: AsRef<Path>>(path: P) -> Result<()> {
```

Comment:
And the helper to avoid this issue was *right here all along*!

Reply:
Haha, my first pass was just replacing the unwrap with `let _ = ...` but then deciding I should probably only ignore NotFound, only to look down and see the exact helper I needed :P